attack_range.conf

```diff
@@ -10,15 +10,15 @@ log_level = INFO
 
 
 [range_settings]
-key_name = attack-range-key-pair
+key_name = attack-range-shannond-oregon
```

Comment:
This file needs to be cleared up.

Reply:
Oops.  Reverting the keys back to original attack-range.conf ones.